openedx2zim/xblocks_extractor/Video.py

```diff
@@ -103,17 +97,18 @@ def download(self, c):
             if subs_lang == {}:
                 subs_lang = self.xblock_json["student_view_data"]["transcripts"]
 
-        if self.scraper.convert_in_webm:
+        if self.scraper.video_format == "webm":
             video_path = self.output_path.joinpath("video.webm")
         else:
             video_path = self.output_path.joinpath("video.mp4")
         if not video_path.exists():
             if youtube:
```

Comment:
Why do we still have that `if youtube` ? I though the downloader was making the download decision now…

Reply:
It's because we need that prepare_url step for instance videos. I agree that this should not be there. However, I think that we shall refactor that at the time of cleaning and refactoring the long functions.